tests/modules/srst2/srst2/test.yml

```
@@ -0,0 +1,49 @@
+- name: srst2 srst2 test_srst2_srst2_exit #Testing pipeline exit when not meta.db
```

```suggestion
- name: srst2 srst2 test_srst2_srst2_exit #Testing pipeline exit when not meta.db
  command: nextflow run tests/modules/srst2/srst2 -entry test_srst2_srst2_exit -c tests/config/nextflow.config
```
The `command` is missing so pytest workflow doesn't have anything to run.

---

@Emiller88 Looks like it I got it to work, thanks for the fix! Let me know if anything else is required.